src/memoryModelsRepo/setIdentifierPersona.ts

```diff
@@ -10,7 +10,7 @@ export default (config: Config) => {
     organisation,
     persona,
     id,
-    locked,
+    locked = false,
```

Comment:
I don't think we need the option here at all because we know we're setting the persona.